app/models/items/donation.rb

```diff
@@ -3,6 +3,12 @@
 
 class Donation < Item
 
+  # only possible donation frequencies (either one-time, or recurring at the specified interval) 
+  @@one_time_donation = 'One Time'
```

Comment:
rails classes should not use class variables as rails is non-reentrant.  is this really a constant?  if so define it as a constant within the class, eg `ONE_TIME_DONATION`, and in code you can refer to it as `Donation::ONE_TIME_DONATION`, etc.  only use a variable if it actually can vary, and don't use class level variabls

Reply:
changed